mitodl/micromasters
# Fix stale version of yarn install

#### Pre-Flight checklist


#### What are the relevant tickets?
N/A

#### What's this PR do?
Discovered in #5331 

#### How should this be manually tested?
- Tests, particularly selenium, should pass
- Run the app, ensure it still builds and runs correctly for you.
